### ISSUE TITLE
Talents bar disappears

### ISSUE THREAD

**Issue by ColdDead** [space-wizards/space-station-14#24089]
The talent bar just disappears, I don't know why. Passier more often than borg

Could have something to do with the switching of the modules, but I'm not sure

Since then there has been no changelog, the problem has been recurring, I looked at Github and didn't see anyone reporting it.

![20240114184958_1](https://github.com/space-wizards/space-station-14/assets/38980900/ddb24e35-7925-4709-a2b0-5c574bf80fd0)

**Comment by deltanedas** [comment]
resetallents command should fix it, iirc its a pvs issue

**Comment by ColdDead** [comment]
Yeah but is not only 1 time is happen one shift 20 times or something, i hope a fix coming for it

**Comment by Pok27** [comment]
I wasn't able to replicate this, so I assume it's already been fixed, or we need more detailed steps to replicate it
